src/main/java/com/box/sdk/BoxSignRequestSigner.java

```diff
@@ -337,6 +337,94 @@ static BoxSignRequestInputType fromJSONString(String jsonValue) {
         }
     }
 
+    /**
+     * Represents a content type of input.
+     */
+    public enum BoxSignRequestInputContentType {
+        /**
+         * Initial content type
+         */
+        Initial("initial"),
+        /**
+         * Stamp content type
+         */
+        Stamp("stamp"),
+        /**
+         * Signature content type
+         */
+        Signature("signature"),
+        /**
+         * Company content type
+         */
+        Company("company"),
+        /**
+         * Title content type
+         */
+        Title("title"),
+        /**
+         * Email content type
+         */
+        Email("email"),
+        /**
+         * Full name content type
+         */
+        FullName("full_name"),
+        /**
+         * First name content type
+         */
+        FirstName("first_name"),
+        /**
+         * Last name content type
+         */
+        LastName("last_name"),
+        /**
+         * Text content type
+         */
+        Text("text"),
+        /**
+         * Date content type
+         */
+        Date("date"),
+        /**
+         * Checkbox content type
+         */
+        Checkbox("checkbox");
+
+        private final String jsonValue;
+        BoxSignRequestInputContentType(String jsonValue) {
+            this.jsonValue = jsonValue;
+        }
+        static BoxSignRequestInputContentType fromJSONString(String jsonValue) {
```

Comment:
Maybe let's use switch which is not used in SDK and I have no idea why...
And we should provide details about the faulty value

```java
static BoxSignRequestInputContentType fromJSONString(String jsonValue) {
    switch (jsonValue) {
        case "initial":
            return Initial;
        case "stamp":
            return Stamp;
        case "signature":
            return Signature;
        case "company":
            return Company;
        case "title":
            return Title;
        case "email":
            return Email;
        case "full_name":
            return FullName;
        case "first_name":
            return FirstName;
        case "last_name":
            return LastName;
        case "text":
            return Text;
        case "date":
            return Date;
        case "checkbox":
            return Checkbox;
        default:
            throw new IllegalArgumentException(
                format("The provided JSON value '%s' isn't a valid BoxSignRequestInputContentType.", jsonValue)
            );
    }
}
```